code/game/machinery/cryopod.dm

```diff
@@ -198,7 +202,15 @@ GLOBAL_LIST_EMPTY(cryopod_computers)
 		if(!control_computer_weakref)
 			control_computer_weakref = cryo_find_control_computer(src, urgent = TRUE)
 
-		despawn_occupant()
+	if(mob_occupant.mind)	//SPLURT Addition - Ghostrole cryoing
```

Comment:
Very fair. Certainly won't deal with ghostroles that just quit and don't bother using the cryo sleeper. 

But that'd require a different implementation signalling the original sleeper somehow which is a touch beyond me this year.

Should work just fine in its limited scope, and not break autocryoing since this check happens before it even reaches the proc autocryo also calls. 

Cheers for taking a look at it all! 

But incidentally autocryo also uses occupant.mind to clear job slots. Wouldn't the mind still exist, but the client be erased?